glib-macros/tests/value_delegate_derive.rs

```diff
@@ -1,9 +1,10 @@
-use glib::{value::FromValue, HasParamSpec, StaticType, ToValue};
+use glib::{value::FromValue, StaticType, ToValue};
 
 #[test]
 fn higher_level_types() {
     #[derive(Debug, glib::ValueDelegate)]
-    pub struct MyVec(Vec<String>);
```

Comment:
Would be useful to have both tested or not?

Reply:
Both nullable and non, or both MyString and MyVec?

Reply:
`MyString` and `MyVec`

Reply:
Yeah, let's keep both. They are good test cases for the ValueDelegate macro, which only had tests in the form of the example documentation.

Reply:
Added both `MyVec` and `MyString`